testing/deployer/topology/images.go

```diff
@@ -29,10 +29,7 @@ func (i Images) LocalDataplaneImage() string {
 		tag = "latest"
 	}
 
-	repo, name, ok := strings.Cut(img, "/")
```

Comment:
Had to update this logic as it was going a bit off the rails when there were 2 slashes, as is the case when using the docker proxy.